pd/Cargo.toml

```diff
@@ -55,7 +55,8 @@ directories = "4.0"
 # Temporarily pin due to https://github.com/penumbra-zone/penumbra/issues/1355
 tokio = { version = "=1.21.1", features = ["full"] }
 tokio-stream = "0.1"
-tokio-util = "0.7"
+# Temporarily pin due to https://github.com/penumbra-zone/penumbra/issues/1977
+tokio-util = "=0.7.4"
```

Comment:
If you bump Tokio to 1.22.0 or greater, then it should work.

Reply:
Many thanks, will try that. There are a few out of date version pins that we may be able to remove, investigating...